tests/test_transpile.py

```diff
@@ -565,6 +565,35 @@ def test_comments(self):
             """with a as /* comment */ ( select * from b) select * from a""",
             """WITH a AS (SELECT * FROM b) /* comment */ SELECT * FROM a""",
         )
+        self.validate(
+            """
+  -- comment at the top
+WITH
+-- comment for tbl1
+tbl1 AS (SELECT 1)
+-- comment for tbl2
+, tbl2 AS (SELECT 2)
+-- comment for tbl3
+, tbl3 AS (SELECT 3)
+-- comment for final select
+SELECT * FROM tbl1""",
+            """/* comment at the top */
+WITH tbl1 /* comment for tbl1 */ AS (
```

Comment:
This looks somewhat inconsistent with how we're generating comments for the subsequent CTEs. Is it simple to have the comments always go after the table name or after the closing parenthesis?

Reply:
It does, tried something quick in my latest commit